tools/privacyidea-token-janitor

```diff
@@ -509,13 +509,18 @@ def find(last_auth, assigned, active, tokeninfo_key, tokeninfo_value,
                 help='Import this PSKC file.')
 @manager.option('--preshared_key_hex', dest='preshared_key_hex',
                 help='The AES encryption key.')
-def loadtokens(pskc, preshared_key_hex):
+@manager.option('--validate_mac', dest='validate_mac',
+                help="How the file should be validated.\n"
+                     "'no_check' : Every token is parsed, ignoring HMAC\n"
+                     "'check_fail_soft' : Skip tokens with invalid HMAC\n"
+                     "'check_fail_hard' : Only import tokens if all HMAC are valid.")
+def loadtokens(pskc, preshared_key_hex, validate_mac):
     from privacyidea.lib.importotp import parsePSKCdata
 
     with open(pskc, 'r') as pskcfile:
         file_contents = pskcfile.read()
 
-    tokens = parsePSKCdata(file_contents, preshared_key_hex)
+    tokens, _ = parsePSKCdata(file_contents, preshared_key_hex, validate_mac)
     success = 0
     failed = 0
     failed_tokens = []
```

Comment:
Should the tokens that were ignored by the import function itself be counted here too?